src/OFS/tests/testCopySupport.py

```diff
@@ -285,32 +285,38 @@ def testPasteMultiSameID( self ):
                                     {'id':'file1', 'new_id':'copy_of_file1'},
                                     {'id':'file2', 'new_id':'copy_of_file2'}])
 
-    def testPasteNoData(self):
+    def assertCopyError(self, callable, error_text, *args):
         from OFS.CopySupport import CopyError
-        with self.assertRaises(CopyError):
-            self.folder1.manage_pasteObjects()
+        try:
+            callable(*args)
+        except CopyError as err:
+            if error_text:
+                self.assertTrue(error_text in str(err))
+        else:
+            self.fail('CopyError not raised.')
```

Comment:
Ugh. Python 2.6 has `assertRaises` as a method -- it just doesn't allow it as a context manager.  ISTM that this would be cleaner as `self.assertRaises(CopyError, callable, *args)` and just drop the assertion about the error text (I don't care for those assertions anyway).

Reply:
@tseaver I know, this is not the nicest code but unfortunately the whole test is about the error text: All three cases result in a `CopyError` but only the error text `Clipboard Error` (Line 310) signals that the the clipboard contents were too big.

An alternative could be to add a dependency to `unittest2` to get more modern unittest assertions but I do not really like it either.